### ISSUE TITLE
Error occured while getting decoy outputs

### ISSUE THREAD

**Issue by keinsell** [mymonero/mymonero-app-js#509]
<img width="494" alt="image" src="https://user-images.githubusercontent.com/41811819/186254029-9f8863f7-bc5b-4eba-8496-42badb25ffae.png">

MacOS Venture, MyMonero 1.3.0

**Comment by hack-r** [comment]
I also have this issue. I am on Linux.

It happened immediately after the balance finally showed up following the hard fork related bug/defect. I assume this is related...

**Comment by KittyKattyKat** [comment]
Same issue on Windows 10. I also get "Error: count cannot be 11"

**Comment by AslanPonies** [comment]
> <img alt="image" width="494" src="https://user-images.githubusercontent.com/41811819/186254029-9f8863f7-bc5b-4eba-8496-42badb25ffae.png">
> 
> MacOS Venture, MyMonero 1.3.0

Hi @keinsell @KittyKattyKat @hack-r @eugeneotto @charleyhine, MyMonero are currently working on the latest Monero hardfork patch. Whilst we are scanning wallets to the current block height where you are able to view your xmr, we are working on the decoy selection issue which relates to send transactions not confirming. We will update you on when the spending function will be enabled. Will keep you updated with progress here.

**Comment by keinsell** [comment]
Meanwhile I've used my mnemonic in Cake Wallet and have sent my funds, so it's no longer critical issue for me.

**Comment by Walker30360** [comment]
It's same on Windows 10. also get "Error: count cannot be 11",MyMonero 1.3.0

**Comment by hack-r** [comment]
Yes all OSs are affected. The MyMonero company replied that they were joking previously about the software being fixed and hopefully it should be fixed soon.

No word on if they are paying us interest on the time-value of money.

**Comment by LivBoss** [comment]
Here is an official announcement from MyMonero:

We are happy to announce that the MyMonero wallet is fully operational again.

We have successfully upgraded the MyMonero software to fall in line with the Monero hard fork.

This is true for all our web-wallet, MacOS and Linux users, as well as those operating on the Android and iOS mobile apps.

You will need to update to the latest software in order to fully operate on the new software.
If you are unsure about this, you can refer to our website - mymonero.com for the latest software versions.

You can now view your XMR balances and transactions, and you can also send funds again.

An important note to our Windows users:
We have successfully completed the hard fork upgrades on the Windows software and although we are very eager to release it, we are awaiting a certificate that will allow us to release this software publicly.
This process is a priority for us and we will announce once our Windows platforms are available again.

Feel free to reach out to us if you have any questions or concerns.

Thank you for your patience.
